server/mlabns/handlers/lookup.py

```diff
@@ -12,54 +13,17 @@
 from google.appengine.ext.webapp import template
 
 
-def _create_af_aware_fqdn(fqdn, address_family):
-    """Adds the v4/v6 only annotation to the fqdn.
-
-    Example:
-        fqdn:       'npad.iupui.mlab3.ath01.measurement-lab.org'
-        ipv4 only:  'npad.iupui.mlab3v4.ath01.measurement-lab.org'
-        ipv6 only:  'npad.iupui.mlab3v6.ath01.measurement-lab.org'
-
-    Args:
-        fqdn: A tool FQDN with no address family specific annotation.
-        address_family: The address family for which to create the FQDN or None
-            to create an address family agnostic FQDN.
-
-    Returns:
-        A FQDN specific to a particular address family, or the original FQDN
-        if an address family is not specified.
-    """
-    if address_family == message.ADDRESS_FAMILY_IPv4:
-        fqdn_annotation = 'v4'
-    elif address_family == message.ADDRESS_FAMILY_IPv6:
-        fqdn_annotation = 'v6'
-    elif not address_family:
-        fqdn_annotation = ''
-    else:
-        logging.error('Unrecognized address family: %s', address_family)
-        return fqdn
-
-    fqdn_parts = fqdn.split('.')
-    fqdn_parts[2] += fqdn_annotation
-
-    return '.'.join(fqdn_parts)
-
-
-def _create_tool_url(fqdn, address_family, http_port):
+def _create_tool_url(fqdn, http_port):
     """Create an HTTP URL for a tool.
 
     Args:
-        fqdn: A tool FQDN with no address family specific annotation.
-        address_family: The address family for which to create the URL or None
-            to create an address family agnostic URL.
+        fqdn: A tool FQDN (already processed by fqdn_rewrite).
         http_port: The HTTP port number part of the URL.
 
     Returns:
-        A tool URL, annotated with the appropriate address family (when
-        available).
+        A constructed tool HTTP URL with protocol and http port.
     """
-    return 'http://%s:%s' % (_create_af_aware_fqdn(fqdn, address_family),
-                             http_port)
+    return 'http://%s:%s' % (fqdn, http_port)
```

Comment:
Similar to the above, the fakewww interface is still the same. We haven't added TLS support to fakewww, so an https URL wouldn't work. This is the fakewww for an NDT-SSL server:

http://ndt.iupui.mlab4.nuq1t.measurement-lab.org:7123/

fakewww only speaks HTTP, not NDT so fakewww's support for TLS is independent of an actual NDT test's support of TLS.